dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -100,7 +112,15 @@ object TankGenerator {
   private def mkPairsAndDistribute(accounts: List[PrivateKeyAccount], pairsFile: Option[File], distributed: Boolean = false): List[AssetPair] = {
     val assets =
       if (Files.notExists(pairsFile.get.toPath)) mkAssets()
-      else readAssetPairs(pairsFile).map(p => { s"${p.getAmountAsset}-${p.getPriceAsset}" }).mkString("-").split("-").toSet.toList
+      else
+        readAssetPairs(pairsFile)
+          .map(p => {
+            s"${p.getAmountAsset}-${p.getPriceAsset}"
+          })
```

Comment:
Small note about style. You can write
```
.map { p =>
  s"${p.getAmountAsset}-${p.getPriceAsset}"
}
```
which requires less symbols.

Reply:
ok